import_export/templates/admin/import_export/import.html

```diff
@@ -5,6 +5,7 @@
 {% load static %}
 
 {% block extrastyle %}{{ block.super }}<link rel="stylesheet" type="text/css" href="{% static "import_export/import.css" %}" />{% endblock %}
+{% block extrahead %}{{ block.super }}{{ media.js }}{% endblock %}
```

Comment:
I notice that `jquery.js` and `jquery.init.js` is already imported.  With your change both files get imported twice.  Is there a reason for this or can we get away with removing the second imports.

![Screenshot from 2022-08-02 20-54-28](https://user-images.githubusercontent.com/6249838/182461794-a27d28b1-2990-4314-b4e7-a94296b97f13.png)

Reply:
Normally it has a uniqness filter, but that did not work because for `DEBUG = True`, it will not work with the `*.min.js` version. That should now be resolved by adding an `if settings.DEBUG` check, and thus use `*.min.js` or the non-minimized version otherwise.